src/Sylius/Component/Core/Model/OrderItemInterface.php

```diff
@@ -56,4 +57,26 @@ public function removeInventoryUnit(InventoryUnitInterface $unit);
      * @return bool
      */
     public function hasInventoryUnit(InventoryUnitInterface $unit);
+
+    /**
+     * @return Collection|OrderItemUnitInterface[]
+     */
+    public function getItemUnits();
+
+    /**
+     * @param OrderItemUnitInterface $itemUnit
+     */
+    public function addItemUnit(OrderItemUnitInterface $itemUnit);
+
+    /**
+     * @param OrderItemUnitInterface $itemUnit
+     */
+    public function removeItemUnit(OrderItemUnitInterface $itemUnit);
+
+    /**
+     * @param OrderItemUnitInterface $itemUnit
+     *
+     * @return bool
+     */
+    public function hasItemUnit(OrderItemUnitInterface $itemUnit);
```

Comment:
I don't know if there is any convention about ordering methods that refer to the same resource (like `get` and `set`), but I prefer queries first (`get`, `has`, `is`) and then commands (`set`, `add`, `remove`). Of course this is just my opinion :)